### PR TITLE
Updated for 1.13.

### PR DESCRIPTION
**Pull request by Catzy44**
I updatedev erything to 1.13

Fixed particles,materials,setdata,etc.
Everything is working ( particles too :) )
Tested it on my mc server - endimc.pl (poland)

I created a SlimeFun pull request too.

### CODE REVIEW

**Comment by TheBusyBiscuit** [comment]
Why did you disable the Updater?

**Comment by TheBusyBiscuit** [comment]
Also it is very important for backwards compatibility to know when the PLAYER_SKULL change to the Material enum was made.

File: src/me/mrCookieSlime/CSCoreLibPlugin/general/Player/PlayerStats.java [hunk 1]
```diff
@@ -49,7 +49,7 @@ public static PlayerStats getStats(Player p) {
 	public long getStatistic(PlayerStat stat) {
 		switch (stat) {
 			case PLAY_TIME_MS: {
-				return Long.valueOf(String.valueOf(Bukkit.getPlayer(uuid).getStatistic(Statistic.PLAY_ONE_TICK) * 50));
+				return Long.valueOf(String.valueOf(Bukkit.getPlayer(uuid).getStatistic(Statistic.PLAY_ONE_MINUTE) * 50));
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/CSCoreLibPlugin/general/Player/PlayerStats.java:52]: Minutes and Ticks are not the same, so the values wouldn't add up in the end.
Please adjust the math of this to reflect that.

File: src/me/mrCookieSlime/CSCoreLibPlugin/general/audio/SoundConfig.java [hunk 2]
```diff
@@ -318,7 +319,7 @@ public boolean onClick(Player p, int slot, ItemStack item, ClickAction action) {
 			}
 		});
 		
-		menu.addItem(4, new CustomItem(new MaterialData(Material.NOTE_BLOCK), "�ePreview", "�rSelected Sound: �b" + cfg.getString(sound + ".enum"), "�rVolume: �b" + cfg.getFloat(sound + ".volume"), "�rPitch: �b" + cfg.getFloat(sound + ".pitch"), "", "�7\u21E8 Click to hear a Preview"));
+		menu.addItem(4, new CustomItem(new MaterialData(Material.NOTE_BLOCK), "�ePreview", "�rSelected Sound: �b" + cfg.getString(sound + ".enum"), "�rVolume: �b" + cfg.getFloat(sound + ".volume"), "�rPitch: �b" + cfg.getFloat(sound + ".pitch"), "", "�7\u21E8 Click to hear a Preview"));
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/CSCoreLibPlugin/general/audio/SoundConfig.java]: Malformed characters in this class.
Use the '&' character instead.

File: src/me/mrCookieSlime/CSCoreLibPlugin/protection/ProtectionManager.java [hunk 3]
```diff
@@ -36,12 +36,12 @@ public ProtectionManager(CSCoreLib cscorelib) {
 		if (cscorelib.getServer().getPluginManager().isPluginEnabled("WorldGuard") && cscorelib.getServer().getPluginManager().isPluginEnabled("WorldEdit")) {
 			registerNewModule("WorldGuard", new WorldGuardProtectionModule());
 		}
-		if (cscorelib.getServer().getPluginManager().isPluginEnabled("Factions")) {
+		/*if (cscorelib.getServer().getPluginManager().isPluginEnabled("Factions")) {
```
Comment (TheBusyBiscuit) [inline src/me/mrCookieSlime/CSCoreLibPlugin/protection/ProtectionManager.java:39]: Why was Factions support removed?

File: src/plugin.yml [hunk 4]
```diff
@@ -1,8 +1,8 @@
 name: CS-CoreLib
-version: ${project.version}
-authors: [mrCookieSlime, John000708, AlexLander123]
+version: 5
```
Comment (TheBusyBiscuit) [inline src/plugin.yml:2]: Do not modify the version like that, let Maven handle that.

**Comment by TheBusyBiscuit** [comment]
I will have to close this PR as you removed a substantial part of the plugin, which is less than ideal for an API...

File: src/me/mrCookieSlime/CSCoreLibPlugin/general/Player/PlayerStats.java (hunk 1 above)
Comment (Catzy44) [reply src/me/mrCookieSlime/CSCoreLibPlugin/general/Player/PlayerStats.java:52]: I will fix this

File: src/me/mrCookieSlime/CSCoreLibPlugin/general/audio/SoundConfig.java (hunk 2 above)
Comment (Catzy44) [reply src/me/mrCookieSlime/CSCoreLibPlugin/general/audio/SoundConfig.java]: I will fix this

File: src/me/mrCookieSlime/CSCoreLibPlugin/protection/ProtectionManager.java (hunk 3 above)
Comment (Catzy44) [reply src/me/mrCookieSlime/CSCoreLibPlugin/protection/ProtectionManager.java:39]: Because API of factions changed

File: src/plugin.yml (hunk 4 above)
Comment (Catzy44) [reply src/plugin.yml:2]: kk

**Comment by Catzy44** [comment]
Ok, i created new commit with changes.

File: src/me/mrCookieSlime/CSCoreLibPlugin/protection/ProtectionManager.java (hunk 3 above)
Comment (TheBusyBiscuit) [reply src/me/mrCookieSlime/CSCoreLibPlugin/protection/ProtectionManager.java:39]: Then update it accordingly or ask someone for help on this but in general we want to keep this as consistent as possible.
And CS-CoreLib functions as a bridge between protection plugins and plugins like Slimefun.
Removing all factions protections without any warning may just be very rude to do.

File: src/me/mrCookieSlime/CSCoreLibPlugin/general/Player/PlayerStats.java (hunk 1 above)
Comment (Catzy44) [reply src/me/mrCookieSlime/CSCoreLibPlugin/general/Player/PlayerStats.java:52]: Done

File: src/me/mrCookieSlime/CSCoreLibPlugin/general/audio/SoundConfig.java (hunk 2 above)
Comment (Catzy44) [reply src/me/mrCookieSlime/CSCoreLibPlugin/general/audio/SoundConfig.java]: Done

File: src/me/mrCookieSlime/CSCoreLibPlugin/protection/ProtectionManager.java (hunk 3 above)
Comment (Catzy44) [reply src/me/mrCookieSlime/CSCoreLibPlugin/protection/ProtectionManager.java:39]: So, send me Factions jar, that you are using.
mail: przemowi2002@gmail.com

File: src/plugin.yml (hunk 4 above)
Comment (Catzy44) [reply src/plugin.yml:2]: Done

File: src/me/mrCookieSlime/CSCoreLibPlugin/protection/ProtectionManager.java (hunk 3 above)
Comment (TheBusyBiscuit) [reply src/me/mrCookieSlime/CSCoreLibPlugin/protection/ProtectionManager.java:39]: I am not using Factions...
I haven't even touched Minecraft since 1.9

Comment (Catzy44) [reply src/me/mrCookieSlime/CSCoreLibPlugin/protection/ProtectionManager.java:39]: Factions jar, that you are using to compile cscorelib.

Comment (TheBusyBiscuit) [reply src/me/mrCookieSlime/CSCoreLibPlugin/protection/ProtectionManager.java:39]: This is a Maven project.
There is no jar...
But I can point you to the pom.xml
https://github.com/TheBusyBiscuit/CS-CoreLib/blob/master/pom.xml

**Pull request closed without merging**